src/main/app/form/validation.ts

```diff
@@ -211,6 +211,11 @@ export const isAddressSelected: Validator = value => {
 };
 
 export const isTextAreaValid: Validator = value => {
+  const matcher = new RegExp(/[<>{}]/);
+  if (matcher.test(value as string)) {
```

Comment:
Does it work for invalid characters put anywhere in the text area? Was it tested?

Reply:
Have shown Vivek the code working over teams.